fury/tests/test_actors.py

```diff
@@ -756,27 +760,45 @@ def test_dots(interactive=False):
 
     npt.assert_equal(scene.GetActors().GetNumberOfItems(), 1)
 
-    extent = scene.GetActors().GetLastActor().GetBounds()
-    npt.assert_equal(extent, (0.0, 1.0, 0.0, 1.0, 0.0, 0.0))
+    arr = window.snapshot(scene)
+    report = window.analyze_snapshot(arr, colors=colors)
+    npt.assert_equal(report.objects, 3)
+    npt.assert_equal(report.colors_found, [False]*3)
```

Comment:
Since you are testing for colors, this test should return true for the colors in the original array. However, the range of the colors in the snapshot is between [0, 255]. Additionally, opacity acts like a percentage of the color, so you could use the opacity column to calculate the corresponding percentage of 255, like this:

```python
expected_colors = np.floor(colors[:, 3] * 255) * colors[:, :3]
report = window.analyze_snapshot(arr, colors=expected_colors)
npt.assert_equal(report.colors_found, [True, True, True])
```